modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/admin/messaging/SynchronizeOccurrenceStatusListener.java

```diff
@@ -35,10 +39,19 @@ private void updateOccurrenceStatus() {
 					.updateOccurrenceStatus(
 						powwowMeetingOccurrence.getOccurrenceId(),
 						OccurrenceStatus.COMPLETED);
+
+				_reindexPowwowMeeting(powwowMeetingOccurrence.getPowwowMeetingId());
```

Comment:
For this specific background job, i think we can save a number of times that trigger the indexer. Should we refactor this way:
- Have a `Set<Long>` to hold all processed `powwowMeetingId`. The set will only keep unique values, so that we can minimize the number of meeting ID. Say, if we process 100 occurrences, but in the end of the day there will be only 5 meetings.
- After finish this loop, have another loop to call the indexer.